cmd/podman/main.go

```diff
@@ -53,6 +53,8 @@ func main() {
 	}
 	logiface.SetLogger(logrusLogger{})
 
+	checkSupportedCgroups()
```

Comment:
Experts, is this the right place? (Also see elsewhere about unreadable `/sys`, AFAICS even `--version` would fail.)

Reply:
Looking at that, `--help`? Remote operation?

_I don’t know what the right place is_, I’m just highlighting the question for other reviewers.